pipelines/dragen_pipelines.py

```
@@ -576,4 +576,30 @@ def get_ploidy_metrics(self):
 
 				run_ploidy_metrics_dict[sample] = parsed_run_ploidy_metrics
 
-		return run_ploidy_metrics_dict
\ No newline at end of file
+		return run_ploidy_metrics_dict
+		
+	def get_cnv_metrics(self):
+		"""
+		Get sample level CNV metrics from dragen cnv metrics file
+		"""
+		
+		results_path = Path(self.results_dir)
+		
+		dragen_cnv_metrics_dict = {}
+		
+		for sample in self.sample_names:
+		
+			cnv_metrics_file = results_path.joinpath(sample).glob(f'*{sample}.cnv_metrics.csv')
+			
+			
+			if len(list(cnv_metrics_file)) == 1:
```

might be better as a try/except to catch silent errors (I'm assuming that the list length should always be 1)